spec/support/database_cleaner.rb

```diff
@@ -0,0 +1,24 @@
+require 'database_cleaner'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.